x/evm/precompile/wasm_test.go

```diff
@@ -500,11 +556,11 @@ func (s *WasmSuite) TestExecuteMultiPartialExecution() {
 	deps := evmtest.NewTestDeps()
 	evmObj, _ := deps.NewEVM()
 
-	wasmContracts := test.SetupWasmContracts(&deps, evmObj, &s.Suite)
+	wasmContracts := test.SetupWasmContracts(&deps, &s.Suite)
```

Comment:
One benefit is that we don't need to pass in `evmObj` to deploy and instantiate wasm contracts anymore. That's useful in test suites where we only want wasm contracts and not EVM.